src/Runners/PHPUnit/ExecutableTest.php

```diff
@@ -280,10 +279,12 @@ protected function assertValidCommandLineLength(string $cmd)
                 throw new \RuntimeException('Command line is too long, try to decrease max batch size');
             }
         }
-            // TODO: Implement command line length validation for linux/osx/freebsd
-            //       Please note that on unix environment variables also became part of command line
-            // linux: echo | xargs --show-limits
-            // osx/linux: getconf ARG_MAX
+        /*
+         * @todo Implement command line length validation for linux/osx/freebsd.
+         *       Please note that on unix environment variables also became part of command line:
+         *         - linux: echo | xargs --show-limits
+         *         - osx/linux: getconf ARG_MAX
+         */
```

Comment:
This appeases phpcs-fixer as it can only be configured with exactly one acceptable phrase for this particular rule.